fixtures/test-duplicate-file-upload/config.toml

```diff
@@ -3,6 +3,8 @@ address="127.0.0.1:8000"
 max_content_length="10MB"
 upload_path="./upload"
 
+[landing_page]
```

Comment:
Is it possible to make this section optional so that we don't have to specify an empty section for disabling the landing page?

Reply:
I was thinking about it when adding it to the test case configs, but then I thought that it was rather highly unlikely that someone would not set either `text` or `file` in production.
Thus I saw the error message for not specifying the section as actually helpful.

But yes, I can do that.

Reply:
I was to hasty. Setting `pub landing_page: Option<LandingPageConfig>,` creates a whole bunch of errors. It seems I have to deal with nested `Option` types and I have no idea how to handle that.
I tried using a similar approach as what you did in main.rs with config.settings, but I couldn't figure it out.

I'll try again tomorrow. But I am also open to any help.

Reply:
I think I managed to make it work, although I am pretty sure there is a better way to do that. 

Additionally I don't know how to fix the tests in the `server.rs`:

```
error[E0609]: no field `text` on type `std::option::Option<LandingPageConfig>`
   --> src/server.rs:365:29
    |
365 |         config.landing_page.text = Some(String::from("landing page"));
    |                             ^^^^

error[E0609]: no field `file` on type `std::option::Option<LandingPageConfig>`
   --> src/server.rs:387:29
    |
387 |         config.landing_page.file = Some(filename.to_string());
    |                             ^^^^

error[E0609]: no field `text` on type `std::option::Option<LandingPageConfig>`
   --> src/server.rs:408:29
    |
408 |         config.landing_page.text = Some(String::from("landing page"));
    |                             ^^^^

error[E0609]: no field `file` on type `std::option::Option<LandingPageConfig>`
   --> src/server.rs:409:29
    |
409 |         config.landing_page.file = Some(filename.to_string());
    |                             ^^^^
```

Reply:
Fixed the test failures, thank you for the implementation!

Reply:
Thanks for the fix. I couldn't figure out how to resolve this!